### ISSUE TITLE
JEI not erasing when exited

### ISSUE THREAD

**Issue by Adam022695** [Darkosto/SkyFactory4Issues#329]
## Steps to Reproduce (for bugs)

1.Search an item and close JEI
2.Search for another item in JEI and see what you last looked for at the front of what you are currently looking for

## Logs

* Client/Server Log:
* Crash Log:

## World Information
 4.1.5
* Preset:
yes
* Prestige: 1

* Modpack Version world created in:

* Additional Content Installed:

## Client Information

* Modpack Version:4.0.1
* Java Version:8
* Launcher Used: twitch launcher
* Memory Allocated: 8gb
* Server/LAN/Single Player:
* Optifine Installed:no
* Shaders Enabled:no

## Server Information
* Java Version:
* Operating System:
* Hoster/Hosting Solution:
* Sponge (Non-Vanilla Forge) Server:

**Comment by Adam022695** [comment]
I like how it didnt actually put all the information in that I gave it. JEI when used to search for an item usually clears after being exited. Instead it keeps the item you last looked for the bar hides the fact that it still has the last thing you had searched for.

**Comment by sk2048** [comment]
If I'm reading this correctly, you're searching for something in JEI, logging out of the world and later logging in again (does it still happen if you close the game?), when you log back in your search box is empty but JEI is still behaving as if whatever you entered is in the box?

JEI ordinarily caches your search term so the last thing you look for is remembered even after you close the game. What would be unusual is if it wasn't showing you.

Which version of Java is it that you're using exactly? Should be a number like 1.8.0_201

**Comment by Adam022695** [comment]
According to what you said its not a bug I just thought it was. It happens when you look up an item in jei and close jei not leave the game. Part of the search before doesn't appear on the screen but like the last letter of what you last searched or sometimes it doesnt at all and I end up with my last search in front of what I just wrote

**Comment by Adam022695** [comment]
8.0.2010.9 Sorry took me a minute to find the whole thing

**Comment by sk2048** [comment]
I think this was a misunderstanding, so I'm going to close it, if I'm wrong feel free to reopen.

**Comment by thefunkykoala** [comment]
I can agree with what @Adam022695 said about how when you search for something in JEI and then close the search window, when you re-open it, it will only show a portion of the search text from the last time you had the window open. i.e. I searched for lead and then closed the search, when I reopened JEI it showed a "d" in the search box so I hit backspace once, but when i typed in to search for another item the "lea" would then appear before my new search.
